diff --git a/src/compiler/llvm-to-backend/amdgpu/LLVMToAmdgpu.cpp b/src/compiler/llvm-to-backend/amdgpu/LLVMToAmdgpu.cpp
index 6be898ae..3d10bc35 100644
--- a/src/compiler/llvm-to-backend/amdgpu/LLVMToAmdgpu.cpp
+++ b/src/compiler/llvm-to-backend/amdgpu/LLVMToAmdgpu.cpp
@@ -153,7 +153,7 @@ public:
       Invocation.push_back("-fno-hip-fp32-correctly-rounded-divide-sqrt");
     }
     
-    if(!llvm::StringRef{ClangPath}.endswith("hipcc")) {
+    //if(!llvm::StringRef{ClangPath}.endswith("hipcc")) {
       // Normally we try to use hipcc. However, when that fails,
       // we may have fallen back to clang. In that case we may
       // have to additionally set --rocm-path and --rocm-device-lib-path.
@@ -173,7 +173,7 @@ public:
             << " does not seem to contain key ROCm bitcode libraries such as ockl.bc. It is "
                "possible that builtin bitcode linking is incomplete.\n";
       }
-    }
+    //}
     
     std::string Output;
     if(!getCommandOutput(ClangPath, Invocation, Output))
@@ -209,6 +209,7 @@ public:
 
 LLVMToAmdgpuTranslator::LLVMToAmdgpuTranslator(const std::vector<std::string> &KN)
     : LLVMToBackendTranslator{sycl::jit::backend::amdgpu, KN, KN}, KernelNames{KN} {
+      HIPSYCL_DEBUG_WARNING << "ASDF Concatting\n";
   RocmDeviceLibsPath = common::filesystem::join_path(RocmPath,
                                                      std::vector<std::string>{"amdgcn", "bitcode"});
 }
@@ -299,6 +300,7 @@ bool LLVMToAmdgpuTranslator::translateToBackendFormat(llvm::Module &FlavoredModu
 }
 
 bool LLVMToAmdgpuTranslator::applyBuildOption(const std::string &Option, const std::string &Value) {
+HIPSYCL_DEBUG_WARNING << "ASDF Arg " << Option << " = " << Value << "\n";
   if(Option == "amdgpu-target-device") {
     TargetDevice = Value;
     return true;
@@ -309,6 +311,7 @@ bool LLVMToAmdgpuTranslator::applyBuildOption(const std::string &Option, const s
     RocmPath = Value;
     return true;
   }
+HIPSYCL_DEBUG_WARNING << "ASDF Ignored " << Option << "\n";
 
   return false;
 }

